### ISSUE TITLE
digitize a slide collection

### ISSUE THREAD

**Issue by whit537** [Ambridge/Ambridge#8]
Received from https://www.facebook.com/pat.parra via Facebook message:

> Hi Chad! Know anyone with some expertise to help me get a scanner for slides I bought for the Library up and running? I would like to copy and digitize a collection there from the estate of an Ambridge historian. There are several problems with the Library end. It cost me $1300. I plan to donate it IF they are going to use it. Otherwise, I will just forget about it and return it to Hammacher Schlemmer.

**Comment by BergFulton** [comment]
Yo! I think I could help with this....I think the problem would be getting the library to be down for it. When I visited and asked to look at the local history collection, it seemed like they were less than pleased to be asked. The storage of the current "collection" is...sub optimal? Photographs in acidic sticky binders, photographs being stored on top of radiators, etc. 

However, this would make an amazing senior project or scout project- the digitizing of things isn't rocket science, but there's a bit of a method to the madness. 

The other thought I had was: I have a lot of professors from Pitt's MLIS program in contact with me right now, and they do a lot of archives training. This would be an amazing placement for the right student (given the right software and hardware). I'd love to see this stuff up on Archives.org too. 

I'm happy to help in any way I can, even if it is just getting the right people connected.

**Comment by whit537** [comment]
Thanks so much for jumping in, @BergFulton! :D

I'll reach out to pat.parra and see if we can get her in on the conversation here. :-)

**Comment by whit537** [comment]
@BergFulton I'm expecting to see pat.parra at the Community Engagement meeting this Thursday (see #9). Any chance you'll be there as well? :-)

**Comment by BergFulton** [comment]
@whit537 Yeah, I should be able to make the engagement meeting. Let me talk about it with @pfulton and make sure we can swing it, but count me in unless reports of my demise are credible. :+1:

**Comment by whit537** [comment]
Awesome! If I don't see you there I'll be sure to talk to pat.parra at least (assuming she's there, sounds like she's planning to be) and follow up with you afterwards.
